### DISCUSSION TITLE
Add split screen local multiplayer

### DISCUSSION THREAD

**Discussion by MishterKirby** [ppy/osu#13590] [Ideas]
It would be cool if you could get some friends together and play some osu! with them, but on only one device. And I think that most modes should work fine with a split screen view (the only one needing the most change being osu!). I think this would make it easier for some people to play a quick match with someone, since the other player doesn't have to have osu! installed, or have an account.

**Comment by memikri** [comment]
How would you suggest the second player's input device be handled separately?

**Reply by MishterKirby** [reply]
Well, I haven't done much programming on the input side of things, but I'm assuming it would be possible to (in the case of osu!std) capture the cursor from each pointing device. As for the other modes, we _could_ just have different keybinds for the second player.
